src/screens/DashboardTx.js

```diff
@@ -99,26 +108,110 @@ function DashboardTx() {
     return elements.map(tx => <TxRow key={tx.tx_id} tx={tx} />);
   };
 
-  return (
-    <div className="content-wrapper">
-      <div className="table-responsive">
-        <table className="table" id="tx-table">
-          <thead>
-            <tr>
-              <th className="d-none d-lg-table-cell">Hash</th>
-              <th className="d-none d-lg-table-cell">Timestamp</th>
-              <th className="d-table-cell d-lg-none" colSpan="2">
-                Hash
-                <br />
-                Timestamp
-              </th>
-            </tr>
-          </thead>
-          {renderTableBody()}
-        </table>
+  const renderUi = () => {
+    return (
+      <div className="content-wrapper">
+        <div className="table-responsive">
+          <table className="table" id="tx-table">
+            <thead>
+              <tr>
+                <th className="d-none d-lg-table-cell">Hash</th>
+                <th className="d-none d-lg-table-cell">Timestamp</th>
+                <th className="d-table-cell d-lg-none" colSpan="2">
+                  Hash
+                  <br />
+                  Timestamp
+                </th>
+              </tr>
+            </thead>
+            {renderTableBody()}
+          </table>
+        </div>
       </div>
-    </div>
-  );
+    );
+  };
+
+  const renderNewTables = (content, text, path) => {
+    return (
+      <div style={{ textAlign: 'center' }}>
+        <div>
+          <table className="table-stylezed">
```

Comment:
fix: minor typo. Maybe there are other similar classes with the same fix.
```suggestion
          <table className="table-stylized">
```

Reply:
Fixed as suggested.